config/TwoCounties.yml

```diff
@@ -1,5 +1,5 @@
 microsim:
-  study-area: "TwoCounties"
+  study-area: "two_counties"
```

Comment:
Maybe to simplify between SPC and ASPICS, we can settle on this `lowercase_underscore` style?